docs/running-workflows/launching-workflows/index.md

```diff
@@ -0,0 +1,125 @@
+# Launch workflows from external source
+
+If your analysis workflows are hosted on external web sources such as GitHub, GitLab, Zenodo,
+it is possible to launch your workflows directly from the REANA web interface's `/launch` web page.
+You can also include a badge in your repository that will allow launching the workflow with a single click.
+
+For example, to launch a [RooFit demo workflow](https://github.com/reanahub/reana-demo-root6-roofit) from GitHub on REANA instance at CERN, you can click on the badge below:
+
+[![Launch RooFit workflow from GitHub](https://www.reana.io/static/img/badges/launch-on-reana-at-cern.svg)](https://reana.cern.ch/launch?url=https%3A%2F%2Fgithub.com%2Freanahub%2Freana-demo-root6-roofit)
+
+## Launcher arguments
+
+To create your own launcher URL, you will need to provide some information to REANA instance on how to launch the workflow.
+This takes a form of various arguments that can be passed in URL.
+The complete launch URL looks like this:
+
+```text
+https://<REANA_SERVER_URL>/launch?url=<WORKFLOW_SOURCE_URL>&name=<WORKFLOW_NAME>&specification=<WORKFLOW_SPEC_PATH>&parameters=<WORKFLOW_PARAMETERS>
+```
+
+Where:
+
+- `REANA_SERVER_URL` is an address of the REANA instance that will be used to launch the workflow;
+- `url`, `name`, etc. are arguments that provide information to launch a workflow.
+
+For example, the above badge for RooFit demo workflow is using this URL:
+
+```text
+https://reana.cern.ch/launch?url=https://github.com/reanahub/reana-demo-root6-roofit?parameters={"events": 10000}
```

Comment:
Note that `{"events": 10000}` will have to be URL-encoded.  This string like that would not be a valid URL.  We would also need to speek about how `parameters` is encoded... Probably later down the page. Here we can just have that example.

Reply:
I encoded `parameters` the same way as the URL.

Reply:
Yes, the same way as other URL parameters.  Here is one example:

```python
In [1]: import urllib

In [2]: urllib.parse.quote('{"events": 30000}', safe='')
Out[2]: '%7B%22events%22%3A%2030000%7D'
```

So an of a launcher URL for RooFit demo with different number of events would be: 

- https://reana.cern.ch/launch?url=https%3A%2F%2Fgithub.com%2Freanahub%2Freana-demo-root6-roofit&name=my-roofit-analysis&specification=reana.yaml&parameters=%7B%22events%22%3A%2030000%7D

Reply:
Yes. I did this way.